src/store/ChartStore.js

```diff
@@ -309,7 +309,7 @@ class ChartStore {
      * @param {bool} status if true, measn mobile mode is active
      */
     setIsMobile(status) {
-        this.isMobile = status;
+        this.isMobile = !status ? CIQ.isMobile : status;
```

Comment:
I think it's better to have `CIQ.isMobile` set by the `charts.binary.com` app. I'll set this when I separate out the app. For the time being discard the changes from `index.js` and ChartStore.js`, and I'll have this merged.